### DISCUSSION TITLE
Size parameter for adaptive bed mesh

### DISCUSSION THREAD

**Discussion by coubi64** [OrcaSlicer/OrcaSlicer#1081] [Q&A]
Hi all.

I need to add size parameter to my start gcode but i don't know the parameters name for OrcaSlicer

This example is for SuperSlicer: 

`PRINT_START SIZE={first_layer_print_min[0]}_{first_layer_print_min[1]}_{first_layer_print_max[0]}_{first_layer_print_max[1]}`

What's the good ones for OrcaSlicer?

**Comment by hliebscher** [comment]
is the Same for OrcaSlicer
{first_layer_print_min[0]}   >  xmin
{first_layer_print_max[0]}  > xmax
{first_layer_print_min[1]}  > ymin
{first_layer_print_max[1]} > ymax

**Comment by coubi64** [comment]
Thank you for your answer.

I tried this two ways to use my PRINT_START. They both work. Is there a better one rather another?

`PRINT_START EXTRUDER_TEMP={first_layer_temperature[0]} BED_TEMP={first_layer_bed_temperature[0]} SIZE={first_layer_print_min[0]}_{first_layer_print_min[1]}_{first_layer_print_max[0]}_{first_layer_print_max[1]}`

`PRINT_START EXTRUDER_TEMP=[nozzle_temperature_initial_layer] BED_TEMP=[bed_temperature_initial_layer] SIZE={first_layer_print_min[0]}_{first_layer_print_min[1]}_{first_layer_print_max[0]}_{first_layer_print_max[1]}`

**Reply by hliebscher** [reply]
Look at your Gcode for this line and Check the Temperature values
